public/app/features/query/components/QueryEditorRow.tsx

```diff
@@ -480,6 +486,28 @@ export class QueryEditorRow<TQuery extends DataQuery> extends PureComponent<Prop
           />
         )}
         {this.renderExtraActions()}
+        <PluginLinksProvider options={{ extensionPointId: 'grafana/query-editor-row/actions' }}>
```

Comment:
This is only here for testing and as an example. Will remove it before merging as how this should be implemented for Query Library isn't yet decided.